src/app/new-page/component/content-video/content-video.component.html

```diff
@@ -0,0 +1,7 @@
+<ng-container *ngIf="ready && provider === 'YOUTUBE' && videoId">
```

Comment:
So, the `provider` and `videoId` checks aren't necessary with the new parser.

If the video is not a youtube video or is missing a videoId, it will not be included in the content array returned to you.

Reply:
of course it doesn't hurt anything leaving these checks in place

Reply:
I assume the likelihood of us moving away from Youtube to another video platform is very low. If that's the case, I'll remove the checks.

Reply:
@dr-bizz if we ever do add a different hosting platform, we would have to deal with parsing it in the parser first, and it would be locked behind a feature flag.

This means you wouldn't get the different video types until you are ready to handle them

Reply:
Awesome! That's a great to know.